examples/fetch/index.tsx

```diff
@@ -23,11 +23,55 @@ const App = () => {
     }
   };
 
+  async function streamToArrayBuffer(stream: any) {
+    const reader = stream.getReader();
+    while (true) {
+      console.log("await read");
+      const { done, value } = await reader.read();
+      console.log("done", done);
+      if (done) {
+        break;
+      } else {
+        console.log(value.byteLength);
+        // @ts-ignore
+        const text = globalThis.TextCodecHelper.decode(value);
+        setData(text);
+        const currentTime = new Date();
+        console.log(text);
```

Comment:
Can we remove these `console.log`?

It looks a little mess if placed on lynx-website.